Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs

```diff
@@ -32,14 +33,27 @@ public void TryStopManipulation()
         {
             if (manipulatedObject != null && collisionTrigger != null && manipulatedObjCollider != null)
             {
-                if (collisionTrigger.bounds.Intersects(manipulatedObjCollider.bounds))
+                if (!collisionTrigger.bounds.Intersects(manipulatedObjCollider.bounds))
                 {
-                    manipulatedObject.ForceEndManipulation();
+                    return;
+                }
+
+                var manipulationHandler = manipulatedObject.GetComponent<ManipulationHandler>();
```

Comment:
Can we cache these when `manipulatedObject` is set instead of calling `GetComponent` twice every `Update`?

Reply:
Can do